phpreport-report

```diff
@@ -385,11 +395,14 @@ class TextFormatter(object):
         indent = (first_column_size + 2) * ' '  # Enough to account for the day name offset.
         width = 80 - len(indent)
         for content in contents:
-            second_column = textwrap.fill(content[1],
-                                          break_long_words=False,  # Don't break URLs.
-                                          width=width,
-                                          initial_indent=indent,
-                                          subsequent_indent=indent).strip()
+            second_column = os.linesep.join(
+                [textwrap.fill(line,
```

Comment:
I'm not sure I understand what is going on here. Are you wrapping each individual line? Is each line really a paragraph?